content/08.transmission.md

```diff
@@ -255,4 +256,30 @@ As a result, facets of systemic inequality such as mass incarceration of Black A
 
 #### Severity of COVID-19 Following Exposure
 
+Following exposure to SARS-CoV-2, the likelihood that an individual develops COVID-19 and the severity of the disease presentation can be influenced by a number of social factors.
+As discussed above, a number of patient characteristics are associated with the likelihood of severe COVID-19 symptoms.
+In some cases, these trends run counter to those expected given rates of exposure: for example, although women are more likely to be exposed, men are more likely to be diagnosed with, hospitalized from, or die from COVID-19 [@doi:10.1371/journal.ppat.1008570].
+In the case of comorbid conditions and racial/ethnic demographics, however, social factors are highly likely to modulate or at least influence the apparent association between the patient trait and the increased risk from COVID-19.
+In particular, the comorbidities and racial/ethnic correlates of severe COVID-19 outcomes suggest that poverty confers additional risk for COVID-19.
+
+In order to explore the relationship between poverty and COVID-19 outcomes, it is necessary to consider how poverty impacts biology.
+Comorbidities that increase risk for COVID-19, including obesity, type II diabetes, hypertension, and cardiovascular disease, are known to be intercorrelated [@doi:10/bpp37d].
+Metabolic conditions related to heightened inflammation, like obesity, type II diabetes, and hypertension, are more strongly associated with negative COVID-19 outcomes than other comorbid conditions, such as chronic heart disease [@doi:10.2337/dbi19-0023].
+As discussed above, dysregulated inflammation characteristic of cytokine release syndrome is one of the greatest concerns for COVID-19-related death.
+Therefore, it is possible that chronic inflammation characteristic of these metabolic conditions predisposes patients to COVID-19-related death [@doi:10.2337/dbi19-0023].
+The association between these disease and severe COVID-19 outcomes is a concern from a health equity perspective because poverty exposes people to "obesogenic" conditions [@doi:10/gddg84] and is therefore unsurprisingly associated with higher incidence of obesity and associated disorders [@doi:10.5604/12321966.1120608].
```

Comment:
"these disease" -> "these diseases"

Reply:
I know that poverty -> predisposition to obesity is definitely true in the US, but I don't think this is true everywhere. The 2nd paper you cite talks about the US and the UK. May want to specify countries for context.

Reply:
Thank you @SiminaB! I'm proposing that we just add a disclaimer at the beginning of this paragraph since I think all of the sources talk about the UK and USA!